### PR TITLE
Fixes apidoc error with Cast Skill

### PR DESCRIPTION
**Pull request by taldin**
Changes Body to Query, changed example from  POST body  (per Blade in the Blacksmith's Guild)

### CODE REVIEW

File: website/server/controllers/api-v3/user.js [hunk 1]
```diff
@@ -408,11 +408,14 @@ const partyMembersFields = 'profile.name stats achievements items.special';
...
  * @apiParamExample {json} Query example:
- *  {
- *     "targetId":"fd427623-9a69-4aac-9852-13deb9c190c3"
- *  }
+ * Cast "Pickpocket" on a task:
+ *  https://habitica.com/api/v3/user/class/cast/pickPocket?targetId=fd427623...
+ *
+ * Cast "Tools of the Trade" on the party:
+ *  https://habitica.com/api/v3/user/class/cast/toolsOfTrade
+ *
```
Comment (librarianmage) [inline website/server/controllers/api-v3/user.js]: There appears to be a trailing space here, which is why the Travis CI test is failing

**Review by librarianmage — CHANGES_REQUESTED** [review]

**Comment by taldin** [comment]
Okay, I think I fixed it.

**Comment by taldin** [comment]
Hmm.  Apparently it's still failing.   I'm not sure how to fix it.

File: website/server/controllers/api-v3/user.js [hunk 2]
```diff
@@ -408,11 +408,13 @@ const partyMembersFields = 'profile.name stats achievements items.special';
  * @apiGroup User
  *
  * @apiParam {String=fireball, mpHeal, earth, frost, smash, defensiveStance, valorousPresence, intimidate, pickPocket, backStab, toolsOfTrade, stealth, heal, protectAura, brightness, healAll} spellId The skill to cast.
- * @apiParam (Body) {UUID} targetId Query parameter, necessary if the spell is cast on a party member or task. Not used if the spell is case on onesself or the user's current party.
+ * @apiParam (Query) {UUID} targetId Query parameter, necessary if the spell is cast on a party member or task. Not used if the spell is case on onesself or the user's current party.
```
Comment (Alys) [inline website/server/controllers/api-v3/user.js]: @taldin The test failure might be just a timeout. Can you please change this:
"Query parameter, necessary if the spell is cast on a party member or task. Not used if the spell is case on onesself or the user's current party."
to this:
"Query parameter, necessary if the skill is cast on a party member or task. Not used if the skill is cast on the user or the user's current party."
which will fix a couple of problems with the text and will also restart the tests. :)

**Comment by taldin** [comment]
Well, it seems to be fixed now!

**Comment by librarianmage** [comment]
\o/

**Comment by taldin** [comment]
Hrm.  I resolved the conflict with the develop branch and now it's failing the test again.

**Comment by TheHollidayInn** [comment]
Noted towards your Tier! Thanks :D

**Comment by taldin** [comment]
Thank ya!   Was there something I could have done to resolve that coveralls error?

**Comment by TheHollidayInn** [comment]
Nah, just a small variation here. You didn't add code so you are good.

**Pull request merged**